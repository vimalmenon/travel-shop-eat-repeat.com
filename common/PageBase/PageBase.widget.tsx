import { Footer } from './Footer';
import { Header } from './Header';

export const PageBase: React.FC = ({ children }) => {
  return (
    <div>
      <Header />
      {children}
      <Footer />
    </div>
  );
};
