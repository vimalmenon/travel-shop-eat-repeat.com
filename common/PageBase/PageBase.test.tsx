import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import { PageBase } from './PageBase.widget';

describe('common > <PageBase> Component', () => {
  describe('Snapshot Testing', () => {
    it('<PageBase>', () => {
      const wrapper = shallow(<PageBase />);
      expect(toJson(wrapper)).toMatchSnapshot();
    });
  });
});
