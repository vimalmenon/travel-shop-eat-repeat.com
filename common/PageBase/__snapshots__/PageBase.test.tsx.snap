// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`common > <PageBase> Component Snapshot Testing <PageBase> 1`] = `
<div>
  <Header />
  <Footer />
</div>
`;
